src/libraries/Common/src/System/Security/Cryptography/RSASecurityTransforms.cs

```diff
@@ -18,6 +18,12 @@ public sealed partial class RSASecurityTransforms : RSA, IRuntimeAlgorithm
         {
             private SecKeyPair? _keys;
 
+            // All values are in bits.
+            // 1024 was achieved via experimentation.
+            // 1024 and 1024+8 both generated successfully, 1024-8 produced errSecParam.
+            // 8192 generated successfully, 8192+8 produced errSecParam.
+            private static readonly KeySizes s_legalKeySize = new KeySizes(minSize: 1024, maxSize: 8192, skipSize: 8);
```

Comment:
you're changing file under Common, are you sure you're changing just Apple?

Reply:
Yes, this is Apple specific. It was in Common due to a S.S.C assembly split.